time stamp
uint32 states
float64[] x

